app/views/users/index.html.erb

```diff
@@ -1,4 +1,5 @@
 <h1 class ="usersTitle">List of all users</h1>
+<%= render 'shared/login_as' %>
 <div class= "allUsers">
   <% @users.each do |user| %>
     <%= link_to render(partial: 'shared/user_info', locals: { user: user }), user_path(user), class: "container" %>
```

Comment:
- [ ] Please the list of users cannot render so make sure that you fix this error in your user's controller :pray: 
  ![image](https://github.com/Alejandro-Bernal-M/rails-blog-app/assets/92303891/3f5b25a9-3f65-4986-8295-12df3cfaad12)

Reply:
@Owusu-Desmond Hello, can you explain to me please at what moment did you get that error? Because when I started the server (rails s) it shows the following:
![image](https://github.com/Alejandro-Bernal-M/rails-blog-app/assets/118059775/bf997ca7-f56e-40cd-b9fc-6d17d370f5d2)

Reply:
@Alejandro-Bernal-M I have tested your project again but I am still getting this same error so please kindly check it or reach out to me on Zoom for your suggestions